hunts/src/tagFilter.js

```diff
@@ -0,0 +1,24 @@
+function getDescendants(tableRow) {
+  const allChildren = [tableRow];
+  for (let i = 0; i < allChildren.length; i++) {
+    const current = allChildren[i];
+    Array.prototype.push.apply(allChildren, current.subRows);
+  }
+  return allChildren;
+}
+
+export function addOrRemoveTag(tagList, tag) {
```

Comment:
Can we leave a comment for this func explaining that it essentially toggles the tag's presence in the array? It wasn't super clear to me from the func name, so even something like "Adds the tag if it doesn't exist or removes it if it does" would be great.

Reply:
Thoughts on renaming this to `toggleTag()`?